checkov/sca_image/runner.py

```diff
@@ -272,8 +272,9 @@ def get_license_statuses(self, packages: list[dict[str, Any]]) -> List[_LicenseS
             ]
             return license_statuses
         except Exception as e:
-            logging.error(f"failing when trying to get licenses-violations: {e}", exc_info=True)
-            return []
+            error_message = "failing when trying to get licenses-violations. it is apparently some unexpected" \
+                            f"connection issue. please try later. in case it keep happening. please report. error is:\n{e}"
+            raise Exception(error_message)
```

Comment:
```suggestion
            error_message = "failing when trying to get licenses-violations. it is apparently some unexpected" \
                            "connection issue. please try later. in case it keep happening. please report"
            raise Exception(error_message) from e
```
this is how you can do error chaining

Reply:
fixed